packages/zipkin/index.d.ts

```diff
@@ -61,6 +62,11 @@ declare namespace zipkin {
     recordLocalAddr(inetAddress: InetAddress): void;
     recordBinary(key: string, value: boolean | string | number): void;
     writeIdToConsole(message: any): void;
+    /** Extract propagation ctx from request */
+    extractId(readHeader: <T> (header: string) => option.IOption<T>): void;
```

Comment:
I think I'd prefer to have the extractId to be actually an extractor like the injector.